README.md

```diff
@@ -32,6 +32,17 @@ set :bundled_sidekiq_roles, [:app] # this is the default value
 
 Set this in `config/deploy.rb` to automate the symlink creation, and then use `XSendFilePath /path/to/my/app/shared/bundled_sidekiq/web/assets` in Apache configuration (in Puppet).
 
+### Status checking
+
+Use `cap ENV check_status` to hit the (*e.g.*, [okcomputer](https://github.com/sportngin/okcomputer)-based) status endpoint of your application.
```

Comment:
Can you add something here about this being valuable when you can't use curl due to firewalls?

Reply:
@jcoyne Done, will push soon.

Reply:
Pushed.